acs5countypoverty.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
Hmm taking a look at this, I think jupyter notebooks are really hard to review... I think we should copy the code from the jupyter notebook into a `.py` script before merging. Thoughts?

Reply:
Copied to .py